connector/connector.go

```diff
@@ -41,6 +41,13 @@ type Traces interface {
 	consumer.Traces
 }
 
+// TracesRouter feeds the first consumer.Traces in each of the specified pipelines.
+// The router will create a fanout consumer for the set of pipelines and return a uuid
+type TracesRouter interface {
```

Comment:
@bogdandrutu, one note about this implementation vs exporting `fanoutconsumer`. Here we add a new interface, but it is not a interface that connectors need to implement. The oimplementation is within the internal fanoutconsumer package. The interface just makes it simple for connectors to get consumers without having to worry about proper implementation.